packages/itwin/breakdown-trees/src/test/functionalityProviders/BuildingClipPlanesProvider.test.ts

```diff
@@ -45,6 +46,7 @@ describe("BuildingClipPlanesProvider", () => {
     queryBuildingRangeStub.restore();
     createRange3DPlaneStub.restore();
     TestUtils.terminateUiFramework();
+    BreakdownTrees.terminate();
```

Comment:
shouldn't the `initialize` call be somewhere around too?

Reply:
added `initialize` to the before part of the test